ivy/functional/backends/jax/random.py

```diff
@@ -57,7 +57,13 @@ def random_uniform(
     else:
         RNG_, rng_input = jax.random.split(_getRNG())
         _setRNG(RNG_)
-
+    if dtype in [jnp.int32, jnp.int64]:
+        return to_device(
+            jax.random.uniform(
+                rng_input, shape, minval=low, maxval=high, dtype=jnp.float32
+            ),
+            device,
+        ).astype(dtype)
```

Comment:
In those parts, I meant that instead of having this line inside an if condition, it should be the default return behavior instead of L#67 to L#70. You'll always cast to `dtype` regardless if it's an `int` or not